### PR TITLE
Dementia Prediction Model

### PR DESCRIPTION
**Pull request by SimranShaikh20**
### Description
Added new folder 

### Issue Resolved
issue no #208 

### Changes Made
Added folder including ,andatory files 

### Checklist

- [x] My code follows the current [project structure](https://github.com/yashasvini121/predictive-calc/blob/master/docs/project-structure.md)
- [x] I have thoroughly reviewed and updated the `requirements.txt` file to include any new packages
- [ ] The `predict.py` file includes a properly implemented `model_details()` function, or the notebook contains this function to print a detailed model report. **The model will not be accepted without this function**, as it is essential for generating the necessary model details.
- [x] I have added relevant tests (if necessary).
- [x] I have added comments in the code where needed.
- [ ] This PR is submitted under **Hacktoberfest**.
- [x] This PR is submitted under **GirlScript Summer of Code (GSSoC-Extd)**.

@yashasvini121 kindly review my PR if any conflicts then let me know !

### CODE REVIEW

File: models/Dementia Prediction Model/dementia-prediction-using-different-ml-model.ipynb
Comment (yashasvini121) [inline models/Dementia Prediction Model/dementia-prediction-using-different-ml-model.ipynb:1]: Missing the `model.py`, `predict.py`, page for the model etc for the **Dementia Prediction Model**.
Please refer the `docs/project-structure.md` and `docs/tutorial.md`

File: models/Pre Owned Car Price Predictor/README.md
Comment (yashasvini121) [inline models/Pre Owned Car Price Predictor/README.md:1]: Follow the correct project structure and create a page for your model

**Review by yashasvini121 — CHANGES_REQUESTED** [review]
Although you've added 3 models, they currently lack the necessary project structure and importantly a **form page**.
Please refer to `docs/tutorial` and `docs/project-structure` for guidance. Feel free to reach out if you have any questions.

**Comment by SimranShaikh20** [comment]
@yashasvini121 so we have to also add streamlit app right?

**Pull request closed without merging**